<launch>
  <node name="urg_node" pkg="urg_node" type="urg_node" output="screen">
    <param name="ip_address" value="192.168.0.11"/>
    <param name="ip_port" value="192.168.0.11"/>
    <param name="angle_min" value="-1.5707963"/>
    <param name="angle_max" value="1.5707963"/>
    <param name="frame_id" value="/lidar_link"/>
    <remap from="most_intense" to="/lidar_scan"/>
  </node>
  <!--node pkg="tf" type="static_transform_publisher" name="lidar_link_broadcaster" args="0 0 0 0 0 0 1 base_link lidar_link 100" /-->
</launch>
